sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -95,7 +95,13 @@ def tearDown(self):
         os.remove(path)
     self._temp_files = []
 
-  def _write_data(self, directory, prefix, codec, count, sync_interval):
+  def _write_data(
+      self,
+      directory=None,
+      prefix=None,
+      codec=None,
+      count=None,
+      sync_interval=None):
```

Comment:
is this necessary if the arguments are mandatory? Are you seeing a failure with this?

Reply:
Yes, they throw `no-value-for-parameter` warnings.